mods/bones/locale/bones.uk.tr

```diff
@@ -0,0 +1,8 @@
+# textdomain: bones
+Bones=Кістки
+@1's old bones=Старі кістки @1
+@1 died at @2.=@1 помер в @2.
+@1 died at @2, and dropped their inventory.=@1 помер в @2 та втратив вміст свого інвентарю.
```

Comment:
Google Translate translates this as "... and lost the contents of his inventory ...", which might be misleading (dropping = turned to items, lost = gone forever)?

Reply:
I clarified the translation of this proposal not to create inaccuracies. You are right, gamer can interpret this  ambiguously.